extension/registry/holixon/holixon-axon/src/main/avro/holixon.registry.AvroSchemaRegistered-v1.avsc

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
I kept my other avsc files in `resources/avro`. Let's discuss pro/cons of having an additional src dir and then use one approach for all schema files.